app/uk/gov/hmrc/ivorchestration/connectors/AuthConnector.scala

```diff
@@ -16,14 +16,13 @@
 
 package uk.gov.hmrc.ivorchestration.connectors
 
-import com.google.inject.Inject
 import uk.gov.hmrc.auth.core.PlayAuthConnector
-import uk.gov.hmrc.ivorchestration.config.AuthServiceConfiguration
 import uk.gov.hmrc.http.HttpClient
+import uk.gov.hmrc.ivorchestration.config.AppConfig
 
+import javax.inject.{Inject, Singleton}
 
-class AuthConnector @Inject()(val http: HttpClient) extends PlayAuthConnector with AuthServiceConfiguration {
-  import authConf._
-
-  lazy val serviceUrl: String = s"$protocol://$host:$port"
+@Singleton
+class AuthConnector @Inject()(val http: HttpClient, app: AppConfig) extends PlayAuthConnector {
```

Comment:
Just checking - do we still need the auth connector in the backend here?  What is it used for?

Reply:
Yes, seems like we still need the auth check for the backend.
Clients might call the backend directly, so for safety I think we should keep it.
https://kibana.tools.production.tax.service.gov.uk/app/r/s/HoRi7